neat_ml/opencv/detection.py

```diff
@@ -0,0 +1,252 @@
+from pathlib import Path
+from typing import Any, Dict, Tuple, Sequence
```

Comment:
I think there's no need to use `Dict` and `Tuple` from `typing` in modern Python (I think pseudo-deprecated since Python `3.9` because you can just use the actual native types).